cfpb/cfgov-refresh
# Careers 2.0: preliminary layout/style changes

Basic layout/style changes to careers pages

## Changes

- __Careers landing page:__ increase spacing in current openings list; move dates into links so they can display as jump links on mobile; style "View all job openings" link as button
- __Job announcement page:__ Move social media links to sidebar; add link to pay grade table; remove "Interested in applying" button 

## Testing

1. Check out branch
2. Verify changes on careers landing & job opening pages

## Screenshots


### Landing page
<img width="1251" alt="screen shot 2017-12-15 at 11 07 13 am" src="https://user-images.githubusercontent.com/778171/34057519-226210be-e18c-11e7-880f-4362dddb5e28.png">

### Landing page links on mobile
<img width="379" alt="screen shot 2017-12-15 at 11 06 49 am" src="https://user-images.githubusercontent.com/778171/34057524-244399fc-e18c-11e7-86c0-b53003cf9d73.png">

### Job posting page, top
<img width="1184" alt="screen shot 2017-12-15 at 11 21 21 am" src="https://user-images.githubusercontent.com/778171/34057507-161603f6-e18c-11e7-8d8a-5e3c3cb38f9a.png">

### Job posting page, social media links removed from apply section
<img width="1156" alt="screen shot 2017-12-15 at 11 21 31 am" src="https://user-images.githubusercontent.com/778171/34057510-1835a6c8-e18c-11e7-9403-50c25be17937.png">

## Checklist
